src/aleph_client/commands/instance/__init__.py

```diff
@@ -140,14 +140,14 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
 
     available_hypervisors = {
         HypervisorType.firecracker: {
-            "Ubuntu 22": settings.UBUNTU_22_ROOTFS_ID,
-            "Debian 12": settings.DEBIAN_12_ROOTFS_ID,
-            "Debian 11": settings.DEBIAN_11_ROOTFS_ID,
+            "ubuntu22": settings.UBUNTU_22_ROOTFS_ID,
+            "debian12": settings.DEBIAN_12_ROOTFS_ID,
+            "debian11": settings.DEBIAN_11_ROOTFS_ID,
```

Comment:
I'm assuming you changed this because it's easier to pass as an argument?

Reply:
Yes and it was not working on my side...